### PR TITLE
feat: output IPs when etcd needs to be bootstrapped

### PR DESCRIPTION
**Pull request by rsmitty**
This PR solves a small UX issue where it's tought to know where to
actually send the talosctl bootstrap command when the VM or server gets
a DHCP address and we don't have access to the router to see what IP it
got.

Signed-off-by: Spencer Smith <spencer.smith@talos-systems.com>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/4730)

### CODE REVIEW

**Comment by rsmitty** [comment]
Screenshot from 2021-12-22 16-06-41

File: internal/app/machined/pkg/system/services/etcd.go [hunk 1]
```diff
@@ -368,9 +368,21 @@ func buildInitialCluster(ctx context.Context, r runtime.Runtime, name, ip string
 		)
 
 		if time.Since(lastNag) > 30*time.Second {
-			log.Printf("etcd is waiting to join the cluster, if this node is the first node in the cluster, please run `talosctl bootstrap`")
-
 			lastNag = time.Now()
+
+			log.Printf("etcd is waiting to join the cluster, if this node is the first node in the cluster, please run `talosctl bootstrap` against one of the following IPs:")
```
Comment (smira) [inline internal/app/machined/pkg/system/services/etcd.go:373]: I like the way it is now, if we want more compact output, we can do `log.Printf("etcd ... following IPs: %s", ips)`

**Review by smira — APPROVED** [review]

File: internal/app/machined/pkg/system/services/etcd.go (hunk 1 above)
Comment (smira) [reply internal/app/machined/pkg/system/services/etcd.go:373]: this should print something like `[127.0.0.1 10.0.0.5]`

Comment (rsmitty) [reply internal/app/machined/pkg/system/services/etcd.go:373]: yeah, I liked the grouping better. updated a bit:

Screenshot from 2021-12-23 09-11-33

**Review by Unix4ever — APPROVED** [review]

File: internal/app/machined/pkg/system/services/etcd.go [hunk 2]
```diff
@@ -368,9 +368,19 @@ func buildInitialCluster(ctx context.Context, r runtime.Runtime, name, ip string
...
+
+			// we "allow" a failure here since we want to fallthrough and attempt to add the etcd member regardless of
+			// whether we can print our IPs
+			currentAddresses, addrErr := r.State().V1Alpha2().Resources().Get(ctx, resource.NewMetadata(network.NamespaceName, network.NodeAddressType, network.NodeAddressCurrentID, resource.VersionUndefined))
+			if addrErr != nil {
+				log.Printf("error getting node addressesasdfasdf: %s", addrErr.Error())
```
Comment (smira) [inline internal/app/machined/pkg/system/services/etcd.go:379]:
```suggestion
				log.Printf("error getting node addresses: %s", addrErr.Error())
```

:rofl:

**Review by smira — APPROVED** [review]

File: internal/app/machined/pkg/system/services/etcd.go (hunk 2 above)
Comment (rsmitty) [reply internal/app/machined/pkg/system/services/etcd.go:379]: Oh good grief. Will fix and repush

**Review by smira — APPROVED** [review]

**Review by smira — APPROVED** [review]

**Pull request merged**